static/js/settings_account.js

```diff
@@ -280,6 +280,27 @@ export function initialize_custom_user_type_fields(
     return user_pills;
 }
 
+export function initialize_bot_owner(element_id, bot_id) {
+    const user_pills = new Map();
+    const bot = people.get_by_user_id(bot_id);
+    const bot_owner = people.get_bot_owner_user(bot);
+    // Bot owner's pill displaying on bots full profile modal.
+    if (bot_owner) {
+        const $pill_container = $(element_id)
+            .find(
+                `.bot_owner_user_field[data-field-id="${CSS.escape(
+                    bot_owner.user_id,
+                )}"] .pill-container`,
+            )
+            .expectOne();
+        const pills = user_pill.create_pills($pill_container);
+
+        user_pill.append_user(bot_owner, pills);
+        user_pills.set(bot_owner.user_id, pills);
+    }
+    return user_pills;
+}
```

Comment:
Why is this function in this file?

Reply:
I think this belongs in `static/js/user_profile.js`. 

The function above it should possibly also move to some place like `profile_fields.js`

Reply:
`settings_profile_fields.js`?

Or create a new file named `profile_fields.js` and move it there?